packages/core/helper-plugin/lib/src/components/FilterListURLQuery/tests/index.test.js

```diff
@@ -13,6 +13,36 @@ import { createMemoryHistory } from 'history';
 import qs from 'qs';
 import FilterListURLQuery from '../index';
 
+jest.mock('react-intl', () => {
+  const reactIntl = jest.requireActual('react-intl');
+  const intl = reactIntl.createIntl({
+    locale: 'en',
+    messages: {
```

Comment:
Could you store these messages as a constant above please and then re-use them in the two cases they are used?